setup.py

```diff
@@ -69,7 +69,6 @@
         "setuptools",
         "Zope2",
         "zope.component",
-        "zope.formlib",
```

Comment:
Doesn't this require z3c.form here?

Egads, you have my approval.

Reply:
Fixed